packages/eas-cli/src/credentials/android/actions/BuildCredentialsUtils.ts

```diff
@@ -17,7 +18,10 @@ export function getAppLookupParamsFromContext(ctx: Context): AppLookupParams {
     throw new Error(`You do not have access to account: ${accountName}`);
   }
 
-  const androidApplicationIdentifier = ctx.exp.android?.package;
+  const androidApplicationIdentifier = await getOrConfigureApplicationIdAsync(
```

Comment:
```suggestion
  const androidApplicationIdentifier = getApplicationId(
```

In https://github.com/expo/eas-cli/pull/414 we changed the conception for the iOS counterpart of `getOrConfigureApplicationIdAsync`. We thought that because of the function name it could be tempting to use it in places that shouldn't require user interaction.

Generally, this function makes sense only for managed workflow - we prompt for the bundle id if it's not defined in app.json. At the same time, we assume that users should be able to update it for generic projects on their own. I renamed `getOrConfigureBundleIdentifierAsync` to `ensureBundleIdentifierIsDefinedForManagedProjectAsync`, and changed the logic so it throws an error if someone is trying to call it against a generic project. See https://github.com/expo/eas-cli/pull/414/files#diff-46f2890e055782597613292b82fd7f55fa05d2af3397df51540a0795487e6362R15

Having `getOrConfigureApplicationIdAsync` call here would make it possible that the prompt shows when someone hasn't configured the project yet with `eas build:configure` or `eas build`, and they run the credentials manager. We'd like to limit the number of places where this interactive logic can happen, ideally to `eas build:configure` and `eas build`.   

I think we should do the same for Android so the implementation is symmetric for both platforms. 

Could you please refactor the `getOrConfigureApplicationIdAsync` function so it's similar to `ensureBundleIdentifierIsDefinedForManagedProjectAsync`? It could be here or in another PR.